aws-datastore/src/test/java/com/amplifyframework/datastore/syncengine/SyncProcessorTest.java

```diff
@@ -69,13 +79,18 @@
 public final class SyncProcessorTest {
     private static final long OP_TIMEOUT_MS = TimeUnit.SECONDS.toMillis(2);
     private static final long BASE_SYNC_INTERVAL_MINUTES = TimeUnit.DAYS.toMinutes(1);
+    private static final List<String> SYSTEM_MODEL_NAMES =
+        Observable.fromIterable(SystemModelsProviderFactory.create().models())
+            .map(m -> m.getSimpleName()).toList().blockingGet();
```

Comment:
`m -> m.getSimpleName()` I think you can write `Class::getSimpleName`, instead?

The stream APIs are a little nicer for simple collection mapping like this aren't they? Oh well.

Reply:
Done.